web/src/user_events.ts

```diff
@@ -163,6 +163,14 @@ export const update_person = function update(event: UserUpdate): void {
     }
 
     if ("custom_profile_field" in event) {
+        // We don't have complete data for the user.
+        if (!user.profile_data) {
```

Comment:
Wait, is the bug that `user` existed but `.profile_data` was undefined, or something else? 

In any case, should the `!user` clause above also call this function?

Reply:
Oops! I forgot to `add_user`.

I still don't understand how `profile_data` can be `undefined` here though. Since `make_user` sets it to `{}` which is not `undefined`.

<img width="596" height="401" alt="image" src="https://github.com/user-attachments/assets/1430031a-8be6-4206-ae91-4542f73699f5" />